src/main/java/org/sagebionetworks/bridge/exporter/worker/ExportWorkerManager.java

```diff
@@ -0,0 +1,430 @@
+    }
+
+    /** Export helper, used to handle attachments. */
+    public final ExportHelper getExportHelper() {
+        return exportHelper;
+    }
+
+    /** @see #getExportHelper */
+    @Autowired
+    public final void setExportHelper(ExportHelper exportHelper) {
+        this.exportHelper = exportHelper;
+    }
+
+    /** File helper, used to create and write to TSVs in the file system. */
+    public final FileHelper getFileHelper() {
+        return fileHelper;
+    }
+
+    /** @see #getFileHelper */
+    @Autowired
+    public final void setFileHelper(FileHelper fileHelper) {
+        this.fileHelper = fileHelper;
+    }
+
+    /** Synapse Helper, used for creating Synapse tables and uploading data to Synapse tables. */
+    public final SynapseHelper getSynapseHelper() {
+        return synapseHelper;
+    }
+
+    /** @see #getSynapseHelper */
+    @Autowired
+    public final void setSynapseHelper(SynapseHelper synapseHelper) {
+        this.synapseHelper = synapseHelper;
+    }
+
+    // STUDY INFO AND OVERRIDES
+
+    /**
+     * Convenience method for getting the access team ID for a given study. The access team ID is the team ID that is
+     * allowed to read the exported data in Synapse. This calls through to DynamoDB and is present to maintain
+     * backwards-compatibility and ease of migration.
+     *
+     * @param studyId
+     *         study ID to get the access team ID for
+     * @return access team ID
+     */
+    public long getDataAccessTeamIdForStudy(String studyId) {
+        StudyInfo studyInfo = dynamoHelper.getStudyInfo(studyId);
+        return studyInfo.getDataAccessTeamId();
+    }
+
+    /**
+     * The Synapse project ID to export to. This first checks the task's request for the project override, then falls
+     * back to Dynamo DB.
+     *
+     * @param studyId
+     *         study ID to get the Synapse project ID for
+     * @param task
+     *         export task, which may or may not have a project ID override
+     * @return project ID to export to
+     */
+    public String getSynapseProjectIdForStudyAndTask(String studyId, ExportTask task) {
+        // check the override map
+        Map<String, String> overrideMap = task.getRequest().getSynapseProjectOverrideMap();
+        if (overrideMap != null && overrideMap.containsKey(studyId)) {
+            return overrideMap.get(studyId);
+        }
+
+        // No override map. Fall back to study config (which is cached).
+        StudyInfo studyInfo = dynamoHelper.getStudyInfo(studyId);
+        return studyInfo.getSynapseProjectId();
+    }
+
+    // TASK AND HANDLER MANAGEMENT
+
+    private final Map<String, AppVersionExportHandler> appVersionHandlersByStudy = new HashMap<>();
+    private ExecutorService executor;
+    private final Map<UploadSchemaKey, HealthDataExportHandler> healthDataHandlersBySchema = new HashMap<>();
+    private final Map<String, IosSurveyExportHandler> surveyHandlersByStudy = new HashMap<>();
```

Comment:
I thought at one point you told me you didn't like this form of initialization... if you're cool with it, I prefer over creating a constructor just to init these kinds of collections.